java/spacewalk-java.changes.bisht-richa.steps-progressbar-component

```diff
@@ -0,0 +1 @@
+- Implemented a new StepsProgressBar component for managing RBAC.
```

Comment:
This changelog is for users, not developers, our users will have no idea what a StepsProgressBarIs. :) 

I would coordinate with @cbbayburt on when and how to have changelogs for RBAC stuff.

Reply:
I will add summarized entries for all the packages before the final merge so you can check "No changelog required" and merge without any chlog entries for now.